### ISSUE TITLE
Support: 

### ISSUE THREAD

**Issue by awallplace** [MISP/MISP#8605]
### Support Questions

Greetings sir, I would have a question for you, when we add feed in MISP, it does not update the feed, what is the reason for this? I want my feed to be updated continuously at certain intervals, is there a way to do this? 

### MISP version

2.4.1

### Operating System

ubuntu

### Operating System version

21

### PHP version

IDK

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by righel** [comment]
Hi @awallplace 
Best way to do this is to set up a cronjob in your MISP instance to run the `fetchFeed` command, to achieve that:
1. Edit `www-data` [crontab](https://man7.org/linux/man-pages/man5/crontab.5.html):
    ```
    sudo crontab -u www-data -e
    ```
2. Add the following line at the bottom:
    ```
    0 0 * * * /var/www/MISP/app/Console/cake Server fetchFeed [user_id] [feed_id|all|csv|text|misp]
    ```
    * This way feeds will be fetched every day at midnight, if you want to change the frequency or times, check this website: https://crontab.guru
    * Please replace `[user_id]` and `[feed_id|all|csv|text|misp]`, for example if you want to pull all feeds as the user with id=1, it should look like this:
    ```
    0 0 * * * /var/www/MISP/app/Console/cake Server fetchFeed 1 all
    ```

Hope this helps.

**Comment by awallplace** [comment]
greetings again, yes we know this command, what I want is that the eventid should stay the same and be updated. I don't want it to create a new eventid. I want it to continuously pull current iocs into the eventid.

I have selected fix event as the target in the feed, but it creates a new event every time.  

We ran your command again and it created a new eventid. 

![resim](https://user-images.githubusercontent.com/43138278/191717959-a21e2920-5265-44d0-9c70-9d583eec91eb.png)

Here, although the fixed event is selected in the target event, what is the reason for this? Can you show me the solution?

**Comment by righel** [comment]
Have you tried setting the Target Event Id? 
If the feed provider generates new UUIDs for each new event you have to set that value to an existing event in your instance.

**Comment by awallplace** [comment]
![resim](https://user-images.githubusercontent.com/43138278/191720923-367bff10-0a64-4c52-b68a-5371365f3cfe.png)
I press the save button and it looks like it saves but I see that the change is not saved and therefore it creates a new eventid, do I need to choose something special in the settings? 


![resim](https://user-images.githubusercontent.com/43138278/191721433-d84341aa-2472-4be7-b96b-200b3fe47d77.png)


![resim](https://user-images.githubusercontent.com/43138278/191721594-3404200d-cba7-409c-8962-b1e47a661670.png)
Even if I do this, the changes are not saved. 



![resim](https://user-images.githubusercontent.com/43138278/191721990-5cd3a425-1e80-4527-9085-86ef78e67f90.png)

**Comment by awallplace** [comment]
![resim](https://user-images.githubusercontent.com/43138278/191725107-2a240a6d-8cd0-4ba8-9832-c5db8c6fbbc7.png)
I changed their places and I was able to get the result I wanted, I was very confused. I think there is a bug and their places have changed because now what I want is happening without any problems. 

![resim](https://user-images.githubusercontent.com/43138278/191725321-1c99bd6a-7835-494d-8142-5516facf9b8b.png)

**Comment by awallplace** [comment]
@righel

**Comment by righel** [comment]
@awallplace: Just checked and there was indeed a bug introduced recently, fixed it via af0aef7db7cbccf75945403bc0737b1cdfc6dede in `2.4` branch, pull the latest code and please try again.

Let me know if that fixed the issue.
